setup.py

```diff
@@ -29,14 +29,14 @@
 
 # TODO(b/174880612): reduce dependency resolution search space
 REQUIRED_PACKAGES = [
-    'absl-py>=0.9,<1.1',
+    'absl-py>=0.9,<=1.1',
     'jinja2>=3.1,<3.2',
     'matplotlib>=3.2.0,<4',
     'jsonschema>=3.2.0,<4',
-    'tensorflow-model-analysis>=0.36.0,<0.37.0',
-    'tensorflow-metadata>=1.5.0,<1.6.0',
-    'tfx>=1.5.0,<1.6.0',
-    'ml-metadata>=1.5.0,<1.6.0',
+    'tensorflow-model-analysis>=0.36.0,<0.40.0',
+    'tensorflow-metadata>=1.5.0,<1.9.0',
+    'tfx>=1.5.0,<1.9.0',
+    'ml-metadata>=1.5.0,<1.9.0',
     'dataclasses;python_version<"3.7"',
```

Comment:
maybe we can add `<2.0.0` instead?

Reply:
Updated to use `<2.0.0` instead. Tested with `1.9.0rc0`.